src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,77 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int ARRAY_GROW_COEFFICIENT = 2;
+    private Node<K, V>[] table = new Node[DEFAULT_INITIAL_CAPACITY];
+    private int size;
 
     @Override
     public void put(K key, V value) {
-
+        if (size > table.length * DEFAULT_LOAD_FACTOR) {
+            resize();
+        }
+        int index = getHush(key) % table.length;
```

Comment:
```suggestion
        int index = getIndex(key) % table.length;
```
We don't need that method it makes no sense you just add 17 to all hash codes. Better create method which returns you the position of Node in table